packaging/nnstreamer.spec

```diff
@@ -396,8 +399,10 @@ You may enable this package to use Google Edge TPU with NNStreamer and Tizen ML
 %define enable_tizen -Denable-tizen=true -Dtizen-version-major=0%{tizen_version_major}
 %define enable_api -Denable-capi=true
 %ifarch %arm aarch64
+%if 0%{?nnfw_support}
```

Comment:
add 
```
%else
%define enable_nnfw_runtime -Dnnfw-runtime-support=disabled
```
or keep it
```
%define enable_nnfw_runtime -Dnnfw-runtime-support=audio
```
so that it won't build nnfw if it does NOT exist.

Reply:
For TV product identification, refer to https://review.tizen.org/gerrit/gitweb?p=platform/upstream/tensorflow.git;a=blob;f=packaging/tensorflow.spec;h=51ced94d67123ac2c1e2e9fb718d91c9ba8fba26;hb=refs/heads/tizen
(Line 109)

Reply:
ACK!

Reply:
Becase `%define enable_nnfw_runtime -Dnnfw-runtime-support=disabled` is already defined in 387 line
so I do not add `%else` statement.